util/checks/index.ts

```diff
@@ -223,7 +226,16 @@ export const DEFAULT_CHECK_CONFIGS = {
   vacuum: {
     inefficient_index_phase: {
       enabled: true,
-      threshold_count: 1,
+      settings: {
+        threshold_count: 1,
+      },
+    },
+    unnecessary_table_growth: {
+      enabled: true,
+      settings: {
+        notify_pct: 20,
+        notify_bytes: 10485760,
```

Comment:
This is just ten megabytes? Is that right?

Reply:
Yup! I suppose I could do 10 * 1024 * 1024 actually, easier to read

Reply:
Sorry, I meant that it seems like a fairly aggressive threshold (i.e., I think it might generate issues for a lot of tables with only modest bloat). Did you try this on our staging data? I don't think it's totally unreasonable, it just jumped out at me.

Reply:
10 GB sounds reasonable to me. That would give us just a handful of alerts.

<img width="813" alt="Screenshot 2023-06-09 at 11 34 39 AM" src="https://github.com/pganalyze/pganalyze-docs/assets/688886/159bd2c2-9534-4077-993b-e49530e530e0">

Reply:
For what it's worth, the check (at least as documented--I have not reviewed the code yet) fires when _both_ the size threshold and percentage threshold are met. But 10MB still seems aggressive to me, even considering that.

Reply:
So, thinking about this some more, a lot of bloat on a small table can be a problem, and there are use cases for single-row (or other very small, single-page) tables. Maybe this threshold is okay (as long as it's _both_ 20% and 10MB). I think it's probably still a good idea to sanity-check this against our own data.

Reply:
I just want to clarify what I'm setting as a threshold (as I just found out that my code was looking at the wrong value 🤦 gonna fix later) -- this threshold is the percentage & bytes of the unnecessary table growth, the growth that could have avoided with more vacuums. This means, it's actually totally unrelated to the current bloat size or % of bloat (we have a discussion to add such alert separately, but not for this one).
In other words, even if the current bloat is big, it's possible that it won't create this issue if that bloat was created long time ago and the table is not creating a bloat actively currently (because in that case, adjusting autovacuum setting won't help the situation).

With that, here is the list of current number with our data. Previously, I've seen that some partitioned tables were creating issues (not this time though, as all are under threshold).

* query_stats_35d_20230606, 7.0 %, 5327.38 MB
* schema_table_scan_methods, 10.0 %, 4019.46 MB (likely this and above are having big "could have avoided bytes" due to high inserts activity, which could make this value less accurate)
* schema_index_stats_hourlies_35d_20230609, 1.0 %, 43.73 MB
* query_stats_hourlies_35d_20230607, 7.0 %, 216.91 MB
* schema_table_stats_hourlies_35d_20230609, 1.0 %, 34.85 MB
* query_tagset_keys, 39.0 %, 8.33 MB
* users, 15.0 %, 0.95 MB
* api_keys, 11.0 %, 0.49 MB

In our case, I think threshold is working pretty well. For larger tables, the percentage correctly (?) helps to avoid triggering. For high percentage tables (`query_tagset_keys`), the size is also helping well. I can see that the size can be increased here though, to something like 100MB?